modalities/dom/components/elements/youtube-viewer.js

```diff
@@ -0,0 +1,142 @@
+/*
+@license
+Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+
+import Xen from '../xen/xen.js';
+
+let template = Xen.html`
+  <div id="player"></div>
```

Comment:
Just a nit, but probably this div serves no purpose. I expect the host is a suitable container, unless you tried it and it didn't work. Not worth any changes, just sharing.

Reply:
I didn't try it, I'll follow up later. There's some more stuff that needs to be added to this to make it more useful and robust.